django/api/models/rebate_historical.py

```diff
@@ -0,0 +1,28 @@
+from shortuuid.django_fields import ShortUUIDField
+from django.conf import settings
+from django.db.models import (
+    CharField,
+    IntegerField,
+    DateField,
+    BooleanField,
+    PROTECT,
+    ForeignKey,
+)
+
+from django.core.validators import MinLengthValidator
+
+from django_extensions.db.models import TimeStampedModel
+from django.utils.translation import gettext_lazy as _
+from ..models import GoElectricRebateApplication
+
+
+class rebate_historical(TimeStampedModel):

```

Comment:
We should follow classname convention. This won't just be historical rebates so maybe we can call this class `GoElectricRebate`

`django/api/migrations/0001_initial.py`


We've been deleting this for now and recreating the migrations using

`python manage.py makemigrations api`

in the docker container.


We've then been making a model export in this file:

`django/api/models/__init__.py`